core/forms.py

```diff
@@ -181,9 +181,9 @@ class SearchPagesForm(SearchPagesFormBase):
     proxtext = fields.CharField(label="Words near each other")
     proxdistance = fields.ChoiceField(choices=PROX_CHOICES)
     # misc
-    lccn = fields.CharField(label="LCCN")
+    #lccn1 = fields.CharField(label="LCCN")
```

Comment:
I vote that we cut these comments entirely so as to keep the codebase as uncluttered (ha) as possible.
Does it show up anywhere in a template, from my prior commit adding it?  Because we should make sure that it's cut there, too!